### PR TITLE
[quality][performance] Format performance numbers in the log

### PR DESCRIPTION
**Pull request by kaiyue0329**
Issue: #6531

#### What it does
- formatted the performance numbers in the log to one decimal place.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/core/src/browser/frontend-application.ts [hunk 1]
```diff
@@ -367,9 +367,9 @@ export class FrontendApplication {
         performance.measure(name, startMark, endMark);
         for (const item of performance.getEntriesByName(name)) {
             if (item.duration > 100) {
-                console.warn(item.name + ' is slow, took: ' + item.duration + ' ms');
+                console.warn(item.name + ' is slow, took: ' + Math.round(item.duration * 10) / 10 + ' ms');
```
[the remark this answers is not in the document]
Comment (vince-fugnitto) [reply packages/core/src/browser/frontend-application.ts]: @kaiyue0329 I think this approach would be best, the idea to have a utility method might be too overkill for such a simple fix.

[the remark this answers is not in the document]
Comment (kaiyue0329) [reply packages/core/src/browser/frontend-application.ts]: @vince-fugnitto Ok. I will change it and submit a new PR.

[the remark this answers is not in the document]
Comment (vince-fugnitto) [reply packages/core/src/browser/frontend-application.ts]: No need to re-submit the pull request, amending and force-pushing would be sufficient 😄

**Comment by kittaakos** [comment]
> would not need to re-implement the same code

We could use [`ms`](https://www.npmjs.com/package/ms).

**Pull request merged**